docs/state.md

```diff
@@ -201,6 +201,7 @@ AppStore
     'security.passwords.last-pass-enabled': boolean, // true if the Last password extension should be enabled
     'security.passwords.manager-enabled': boolean, // whether to use default password manager
     'security.passwords.one-password-enabled': boolean, // true if the 1Password extension should be enabled
+    'security.passwords.bitwarden-enabled': boolean, // true if the bitwarden extension should be enabled
```

Comment:
Can you please move this line after the line `'security.passwords.active-password-manager'`, so that properties are ordered alphabetically